TESTS/network/wifi/main.cpp

```diff
@@ -94,18 +94,18 @@ int main()
 {
     return !Harness::run(specification);
 }
-#endif // defined(MBED_CONF_APP_WIFI_UNSECURE_SSID) &&  \
-!defined(MBED_CONF_APP_AP_MAC_UNSECURE)    || \
-!defined(MBED_CONF_APP_MAX_SCAN_SIZE)      || \
-!defined(MBED_CONF_APP_WIFI_CH_UNSECURE)   || \
-!defined(MBED_CONF_APP_WIFI_UNSECURE_SSID)
+#endif // defined(MBED_CONF_APP_WIFI_UNSECURE_SSID) &&
+// !defined(MBED_CONF_APP_AP_MAC_UNSECURE)    ||
```

Comment:
@0xc0170 , I don't think so. This is the closing `#endif` for the `#define` from lines 41-45. I think it makes sense to leave it as close as possible to the opening one.
What we can do is decrease the number of lines, by putting multiple checks into one line, but this will decrease readability...

Reply:
The warning that I am addressing here is that the "comment was multiline". The compiler interprets the "\" at the end of the line as a continuation of the comment, rather than a continuation of a macro. After we remove the "\"s we need to add the new comment indicators at the beginning of each line.